src/libs/OptionsListUtils.ts

```diff
@@ -1490,8 +1490,8 @@ function getValidOptions(
     const loginsToExclude: Record<string, boolean> = {
         [CONST.EMAIL.NOTIFICATIONS]: true,
         ...excludeLogins,
-        // Exclude Manager McTest if user submitted expense or scanned receipt and when selection is made from Create or Submit flow
-        [CONST.EMAIL.MANAGER_MCTEST]: !(!getIsUserSubmittedExpenseOrScannedReceipt() && canShowManagerMcTest && Permissions.canUseManagerMcTest(config.betas)),
+        // Exclude Manager McTest if selection is made from Create or Submit flow
+        [CONST.EMAIL.MANAGER_MCTEST]: !(canShowManagerMcTest && Permissions.canUseManagerMcTest(config.betas)),
```

Comment:
`!getIsUserSubmittedExpenseOrScannedReceipt() || userHasReportWithManagerMcTest`

I think this is more what we'd want, otherwise we may show it to users who dismissed the modal and never sent an expense to the account?

Reply:
Right, makes sense! I will make this a local util so we can check if the report is a Manager McTest report in other methods if necessary